src/core/utils/EventNormalizer.ts

```diff
@@ -207,34 +202,91 @@ export class EventNormalizer {
             }
             const childNodes = clone.origin.childNodes;
             clone.childNodes.forEach((child: Node, index) => {
-                child['orgin'] = childNodes[index];
+                child['origin'] = childNodes[index];
                 addChildOrigin(child as ClonedNode);
             });
         })(clone);
     }
     /**
-     * Process the compiled event by triggering the corresponding events on the
-     * listener and reset the compiling process for the next stack.
+     * Process the simplified events stack and create the compiled event.
+     * Call _processCompiledEvents who process this compiled event.
      *
      * @private
      */
-    _processCompiledEvent(): void {
+    _processSimplifiedEvents(): void {
         // Store and reset compiled event for the next stack.
-        const compiledEvent = this._compiledEvent;
-        this._compiledEvent = null;
+        const simplifiedEvents = this._simplifiedEvents;
```

Comment:
In ES6, wouldn't it be possible to do `const {events, clone} = this._simplifiedEvents` ?